news/test_gui_ca_exp_creator.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* Added pytest code for gui_ca_exp_creator
```

Comment:
This is really great, thanks for doing this!  Since this will go into the changelog at the next release, we only need a single statement that we added tests for the GUI code.  To make the code pass pre-commit CI please put this statement in each news file:

`No news needed: one part of a larger set of tests being added for gui code`

except for one news file where we want to write:
 `Extensive tests for GUI code` 
under Added